css-three-column-layout/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="text-align-center ">
+      <div class="row">
+        <div class="column-full">
+          <header class=" font-size-bigger white-text"> Designed for life science.Optimized for SaMD and Digital Health.
+          </header>
+        </div>
+      </div>
+
+      <div class="column">
+        <div class="column-one-thirds">
```

Comment:
`column` and `column-one-thirds` at once? You should only need one `column` element, not a `column` as a child of a `column`.

Reply:
This note still needs to be addressed. The three `column-one-thirds` elements here should be children of an element with the `row` class. `column` class elements must always be a direct child of an element with the class `row`